ani-cli

```diff
@@ -465,58 +509,53 @@ check_input
 append_history
 open_selection
 
+
+########
+# Loop #
+########
+
 while :; do
-	printf "\n${c_green}Currently playing %s episode ${c_cyan}%d/%d\n" "$selection_id" "$episode" "$last_ep_number"
+	inf "Currently playing $selection_id episode" "$episode/$last_ep_number"
 	if [ "$episode" -ne "$last_ep_number" ]; then
-		printf "$c_blue[${c_cyan}%s$c_blue] $c_yellow%s$c_reset\n" "n" "next episode"
+		menu_line_odd "next episode" "n"
 	fi
 	if [ "$episode" -ne "1" ]; then
-		printf "$c_blue[${c_cyan}%s$c_blue] $c_magenta%s$c_reset\n" "p" "previous episode"
+		menu_line_even "previous episode" "p"
 	fi
 	if [ "$last_ep_number" -ne "1" ]; then
-		printf "$c_blue[${c_cyan}%s$c_blue] $c_yellow%s$c_reset\n" "s" "select episode"
+		menu_line_odd "select episode" "s"
```

Comment:
Is there a way to avoid hardcoding alternation?
![image](https://user-images.githubusercontent.com/82055622/150844824-2af1eccf-17de-4700-9db8-01e090ef01f1.png)

Reply:
I can do that, but it'd require an if-else statement at every option. I don't think it's worth it.

Reply:
Fair enough, if no elegant solution occurs to us over night we'll leave it like that